spec/models/picture_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe Picture, type: :model do
+
+  it { is_expected.to belong_to :user }
+  it { is_expected.to have_many :comments }
+  it { is_expected.to have_many :likes }
+  it { is_expected.to have_attached_file(:image)}
+  it { is_expected.to validate_attachment_content_type( %r(Aimage/.*\Z))}
```

Comment:
Style/PercentLiteralDelimiters: `%r`-literals should be delimited by `{` and `}`. (https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)